api/src/libs/utils.ts

```diff
@@ -98,3 +98,8 @@ export const transformContacts = (contactSet: any[]) => {
 export const generateNamespaceNames = (namespacePrefix: string) => {
   return PROJECT_SET_NAMES.map((n) => `${namespacePrefix}-${n}`);
 };
+
+export const getLicencePlatePostFix = (input: string | undefined): any => {
+  const regexMatch = input ? input.match(/(?<=-).*/) : null;
+  return regexMatch ? regexMatch[0] : "";
+};
```

Comment:
unit test this utility

Reply:
also why does this return ""

Reply:
Add unit test.
Its because I don't want to return null and need to handle this special case when I use it.

Reply:
fair enough